third_task/printSorted/printSorted.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func main() {
+	myMap := map[int]string{
+		5: "think",
+		2: "is",
+		3: "very",
+		1: "Go",
+		4: "strange",
+	}
+	printSorted(myMap)
+}
+
+func printSorted(givenMap map[int]string) {
+	res := []string{}
+	keys := []int{}
+	for k := range givenMap {
+		keys = append(keys, k)
+	}
+
+	sort.Ints(keys)
+
+	for _, k := range keys {
+		res = append(res, givenMap[k])
+	}
+
+	fmt.Println(res)
```

Comment:
We should print map values, not keys.

Reply:
everything is correct, but variable has wrong name.

Reply:
Creating new slice for this purpose not a good idea